bundles/org.smarthomej.binding.notificationsforfiretv/README.md

```diff
@@ -0,0 +1,57 @@
+# Notifications For Fire TV Binding
+
+The Notifications For Fire TV Binding provides support for sending notifications to a Fire TV from rule.
+To use the binding it is necessary to install the free app `Notifications For Fire TV` by Christian Fees on the Fire TV.
+The app is available on the Amazon app store by searching for the name.
+
+## Supported Things
+
+There is only one thing: `notification` which represents a Fire TV.
+
+## Thing Configuration
+
+There are two mandatory parameters `ip` and `host`.
+
+The `ip address` must be a valid ip address to a Fire TV.
+The `port` should be the default port `7676` of the app Notifications For Fire TV.
+
+The other parameters are optional but recommendable.
+
+The `tile` parameter may be the sender of the notification.
+The `duration` defines how long the notifications stay on the screen.
+The `position` is used to set the location of the notification on the screen.
+The `transparency` set the alpha channel of the notification background.
+The `horizontal offset` changes the x coordinate of the notification relative to the `position`.
```

Comment:
```suggestion
The `offsetX` changes the x (horizontal) coordinate of the notification relative to the `position`.
```

Reply:
This is not possible because it represents the name of the part in a multipart. So the developer of Notificatins for Fire TV decided to define the name for the x offset only offset and the name for the y offset offsety. This is why i also defined all the x offset variables only offset and not offsetx.

Reply:
done